classes/02class/exercises/c02-network02/nimmi89/README.md

```diff
@@ -0,0 +1,24 @@
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+|Subnet|Network|Host Range|Usable Hosts|Broadcast|AWS Reserved|
+|-|-|-|-|-|-|
+|subnet-a|10.0.0.0/18 | 10.0.0.0| 10.0.0.4-10.0.63.254 |16379 |10.0.63.255| 10.0.0.1, 10.0.0.2, 10.0.0.3|
+|subnet-b|10.0.64.0/18 |10.0.64.0 |10.0.64.4-10.0.127.254 |16379 |10.0.127.255| 10.0.64.1, 10.0.64.2, 10.0.64.3 |
+|subnet-c|10.0.128.0/18 |10.0.128.0 |10.0.128.4-10.0.191.254 |16379 | 10.0.191.255 |10.0.128.1, 10.0.128.2, 10.0.128.3|
+|subnet-d|10.0.192.0./18 |10.0.192.0 |10.0.192.4-10.0.254.255 |16379 |10.0.255.255| 10.0.192.1, 10.0.192.2, 10.0.192.3|
```

Comment:
The Host range should be all the hosts that are part of the network. Not the hosts available on a AWS VPC with that subnet..
Can you fix your response?

Reply:
Fixed the host range to include all including AWS ones